lib/guardian/post_guardian.rb

```diff
@@ -173,6 +172,11 @@ def can_edit_post?(post)
     false
   end
 
+  def can_edit_all_regular_posts?
```

Comment:
This method and its name is a bit weird. There already exists `can_edit_post?` and this one is `can_edit_all_regular_posts?`. (what is "regular"?)

I believe a better name is `is_in_edit_post_group?` if we were to keep to what it is doing.